brainrender_napari/widgets/atlas_manager_view.py

```diff
@@ -38,7 +38,9 @@ def __init__(self, parent: QWidget = None):
         """
         super().__init__(parent)
 
-        self.setModel(AtlasTableModel(AtlasManagerView))
```

Comment:
This and other changes to `atlas_manager_view` and `atlas_table_model` are _in_ _theory_ sensible (I need to sleep on it) but they are unrelated to this issue and this PR: it's good practice to have only changes for 1 issue / PR. 

In particular, I think in this case, these unrelated changes break our tests. So they should be removed, and possibly be added in a separate PR, without breaking the tests.